tests/scripts/check-generated-files.sh

```diff
@@ -139,6 +139,8 @@ if in_mbedtls_repo; then
     check scripts/generate_query_config.pl programs/test/query_config.c
     check scripts/generate_features.pl library/version_features.c
     check scripts/generate_ssl_debug_helpers.py library/ssl_debug_helpers_generated.c
+    check tests/scripts/generate_test_keys.py tests/src/test_keys.h
```

Comment:
Note: I'm not super sure about this line (my previous comment was only about `test_certs.h`, so I'm asking on slack.

Reply:
`test_keys.h` is used by crypto (`pk.h`, likely some day by a PSA equivalent API), so it belongs in the crypto+TLS section. But that's not really important: the sections in this file date back from before we had planned the framework repository, so they'll need to be revised anyway.

Reply:
So it's not critical. But it would help to get the section right, because the next person to work on this file might be dealing with lost of unfamiliar files, so we should try to give them clues as to what to put where.